validation/security-configs_test.go

```diff
@@ -0,0 +1,167 @@
+	require.NoError(t, err)
+
+	// Portal version `3` is the first version of the `OptimismPortal` that supported the fault proof system.
+	isFPAC := majorVersion >= 3
+
+	if isFPAC {
+		contractCallResolutions = append(contractCallResolutions,
+			resolution{"DisputeGameFactoryProxy", "admin()", "ProxyAdmin"},
+			resolution{"AnchorStateRegistryProxy", "admin()", "ProxyAdmin"},
+			resolution{"DelayedWETHProxy", "admin()", "ProxyAdmin"},
+			resolution{"DelayedWETHProxy", "admin()", "ProxyAdmin"},
+			resolution{"DelayedWETHProxy", "owner()", "ProxyAdminOwner"},
+			resolution{"OptimismPortalProxy", "guardian()", "Guardian"},
+			resolution{"OptimismPortalProxy", "systemConfig()", "SystemConfigProxy"},
+		)
+	} else {
+		contractCallResolutions = append(contractCallResolutions,
+			resolution{"OptimismPortalProxy", "GUARDIAN()", "Guardian"},
+			resolution{"OptimismPortalProxy", "SYSTEM_CONFIG()", "SystemConfigProxy"},
+			resolution{"OptimismPortalProxy", "L2_ORACLE()", "L2OutputOracleProxy"},
+			resolution{"L2OutputOracleProxy", "admin()", "ProxyAdmin"},
+			resolution{"L2OutputOracleProxy", "CHALLENGER()", "Challenger"},
+		)
+	}
+
+	for _, r := range contractCallResolutions {
+		contractAddress, err := Addresses[chainID].AddressFor(r.name)
+		require.NoError(t, err)
+
+		want, err := Addresses[chainID].AddressFor(r.shouldResolveToAddressOf)
+		require.NoError(t, err)
+
+		got, err := getAddressWithRetries(r.method, contractAddress, client)
+		require.NoErrorf(t, err, "problem calling %s.%s", contractAddress, r.method)
+
+		assert.Equal(t, want, got, "%s.%s = %s, expected %s (%s)", r.name, r.method, got, want, r.shouldResolveToAddressOf)
+	}
+
+	// Perform an extra check on a mapping value of "L1CrossDomainMessengerProxy":
+	// This is because L1CrossDomainMessenger's proxy is a ResolvedDelegateProxy, and
+	// ResolvedDelegateProxy does not expose a getter method to tell us who can update its implementations (i.e. similar to the proxy admin role for a regular proxy).
+	// So to ensure L1CrossDomainMessengerProxy's "proxy admin" is properly set up, we need to peek into L1CrossDomainMessengerProxy(aka ResolvedDelegateProxy)'s storage
+	// slot to get the value of addressManager[address(this)], and ensure it is the expected AddressManager address, and together with the owner check within AddressManager,
+	// we now have assurance that L1CrossDomainMessenger's proxy is properly managed.
+	l1cdmp, err := Addresses[chainID].AddressFor("L1CrossDomainMessengerProxy")
+	require.NoError(t, err)
+	actualAddressManagerBytes, err := getMappingValue(l1cdmp, 1, l1cdmp, client)
+	require.NoError(t, err)
+	am, err := Addresses[chainID].AddressFor("AddressManager")
+	require.NoError(t, err)
+	assert.Equal(t,
+		am[:],
+		actualAddressManagerBytes[12:32],
+	)
+}
+
+func TestSecurityConfigs(t *testing.T) {
+	isExcluded := map[uint64]bool{
+		11155421: true, // OP_Labs_Sepolia_devnet_0  (no SystemConfigOwner specified)
```

Comment:
We should follow up on why these are absent? We can probably backfill this information with `cast call`.

Reply:
I'm a bit stuck trying to chase down `AnchorStateRegistryProxy` address for   "sepolia-dev-0/base-devnet-0" and 
  "sepolia-dev-0/oplabs-devnet-0". 
 
I wonder if @clabby knows where to get this information?
  
Seeing as these chains are not covered by the solidity test which I am removing on this PR, I think it is reasonable to chase this down in a followup PR.

Reply:
https://github.com/ethereum-optimism/superchain-registry/issues/264